src/m365/aad/commands/group/Group.ts

```diff
@@ -0,0 +1,23 @@
+export interface Group {
```

Comment:
Recently, we've added a reference to the Microsoft Graph types package (`@microsoft/microsoft-graph-types`), so rather than introducing our own type, let's reference it from there.